From 192837172632b72c0da82ade708bb6b1ed649b8f Mon Sep 17 00:00:00 2001
From: Daniel Drake <dsd@gentoo.org>
Date: Wed, 22 Nov 2006 03:16:13 +0000
Subject: [PATCH] [PATCH] adm8211: Use ieee80211_generic_get_stats

adm8211 can also benefit from the generic get_stats implementation in my
previous mail. I am sending the adm8211 patch separately because adm8211 is
not in Linus' tree.

Signed-off-by: Daniel Drake <dsd@gentoo.org>
Signed-off-by: John W. Linville <linville@tuxdriver.com>
---
 adm8211/adm8211_hw.c |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/adm8211/adm8211_hw.c b/adm8211/adm8211_hw.c
index f95d798..a33d239 100644
--- a/adm8211/adm8211_hw.c
+++ b/adm8211/adm8211_hw.c
@@ -363,12 +363,6 @@ static void adm8211_clear_sram(struct net_device *dev)
 	ADM8211_CSR_WRITE(WEPCTL, reg);
 }
 
-static struct net_device_stats *adm8211_get_stats(struct net_device *dev)
-{
-	struct ieee80211_device *ieee = netdev_priv(dev);
-	return &ieee->stats;
-}
-
 static void adm8211_set_rx_mode(struct net_device *dev)
 {
 	struct adm8211_priv *priv = ieee80211_priv(dev);
@@ -2531,7 +2525,6 @@ static int __devinit adm8211_probe(struct pci_dev *pdev,
 	dev->features |= NETIF_F_LLTX;
 	dev->open = adm8211_open;
 	dev->stop = adm8211_stop;
-	dev->get_stats = adm8211_get_stats;
 	dev->set_multicast_list = adm8211_set_rx_mode;
 	dev->set_mac_address = adm8211_set_mac_address;
 
-- 
1.5.2.1

